builtin/providers/aws/resource_aws_config_config_rule.go

```diff
@@ -110,6 +110,7 @@ func resourceAwsConfigConfigRule() *schema.Resource {
 									"event_source": {
 										Type:     schema.TypeString,
 										Optional: true,
+										Default:  "aws.config",
```

Comment:
@mbfrahry I think we are going to need a migration for this - if people are not setting this value then they may get changes in their plans

Thoughts?